apps/ledger-live-desktop/src/newArch/features/WalletSync/hooks/useWatchWalletSync.ts

```diff
@@ -118,6 +126,16 @@ export function useWatchWalletSync(): WalletSyncUserState {
     [visualPending, walletSyncError, onUserRefresh],
   );
 
+  const watchConfig = useMemo(
+    () => ({
+      notificationsEnabled: featureWalletSync?.params?.watchConfig?.notificationsEnabled,
+      pollingInterval: featureWalletSync?.params?.watchConfig?.pollingInterval,
+      initialTimeout: featureWalletSync?.params?.watchConfig?.initialTimeout,
+      userIntentDebounce: featureWalletSync?.params?.watchConfig?.userIntentDebounce,
+    }),
```

Comment:
```suggestion
    () => featureWalletSync?.params?.watchConfig,
```

we should be able to just do this, no?